libsel4/sel4_arch_include/aarch64/sel4/sel4_arch/constants.h

```diff
@@ -174,7 +174,11 @@ typedef enum {
 #define seL4_LargePageBits 21
 #define seL4_HugePageBits 30
 #define seL4_SlotBits 5
+#if defined(CONFIG_HARDWARE_DEBUG_API) || (defined(CONFIG_ARM_HYPERVISOR_SUPPORT) && defined(CONFIG_ARM_HYP_ENABLE_VCPU_CP14_SAVE_AND_RESTORE))
```

Comment:
Checking for CONFIG_HARDWARE_DEBUG_API and CONFIG_ARM_HYP_ENABLE_VCPU_CP14_SAVE_AND_RESTORE seems enough?